docs/scenes/get-started/set-up-your-environment/instructions/iosPhysicalDevelopmentBuildLocal.mdx

```diff
@@ -44,6 +44,8 @@ Run the following command in your project's root directory:
 
 ### Run the project on your device
 
+Add the `ios.bundleIdentifier` in the **app.json** file in the root directory to a unique value so that Xcode generates the provisioning profile for app signing step.
```

Comment:
```suggestion
1. Add the `ios.bundleIdentifier` in the **app.json** file in the root directory to a unique value so that Xcode generates the provisioning profile for the app signing step.
```